packages/legend-application-studio/src/components/editor/editor-group/function-activator/SnowflakeAppFunctionActivatorEditor.tsx

```diff
@@ -253,6 +271,45 @@ export const SnowflakeAppFunctionActivatorEditor = observer(() => {
               }
             />
           </PanelForm>
+          <PanelForm>
```

Comment:
is this form correct. shouldn't there be an option to add both of these properties as they are initially undefined ?